lib/mix/lib/mix/tasks/app.tree.ex

```diff
@@ -20,6 +20,8 @@ defmodule Mix.Tasks.App.Tree do
     * `--pretty` - use Unicode codepoints for formatting the tree.
       Defaults to true except on Windows.
 
+    * `--dot` - produces a DOT graph description of the application tree.
```

Comment:
I think they deserve merging since they both affect one thing and are mutually exclusive.